session8/src/com/itschool/homework/EX5.java

```diff
@@ -0,0 +1,10 @@
+package com.itschool.homework;
+
+public class EX5 {
+
+    public static void main(String[] args) {
+//        help here, please
```

Comment:
Sure. Here is a way to do it:
```
String text = "such a beautiful day!";

StringBuilder result = new StringBuilder(); // we will build the result into this stringBuilder
boolean isNextCapitalizeNeeded = true; // flag to indicate whether we need to capitalize next char.
// Start with it as true, because we want to capitalize the first letter in the text
for (char c : text.toCharArray()) { // iterate through the characters of the text 
	if (isNextCapitalizeNeeded) {
		result.append(Character.toUpperCase(c)); //capitalize the char if needed and append it to result,
		// then mark the flag to false, so we don't capitalize all the letters
		isNextCapitalizeNeeded = false;
	} else if (Character.isWhitespace(c)) { // in case we have a space,
		// then mark the flag to true because after space we want to capitalize the letter
		isNextCapitalizeNeeded = true;
		result.append(c); // append also the space
	} else {
		result.append(c); // append the rest of letters
	}
}

System.out.println(result); // print the result
```